game-headed/src/main/java/org/triplea/game/client/ui/javafx/screen/RootActionPane.java

```diff
@@ -92,7 +95,9 @@ public void switchScreen(final Screens identifier) {
       case CONTENT:
         exitOverlay.setVisible(false);
         loadingOverlay.setVisible(false);
-        rootPane.setVisible(true);
+        if (root.getChildren().size() == 3) {
```

Comment:
Would it make sense to also hide parts in the other case statement cases? If so, I think a common function to hide all before showing the relevant parts might be a good way to approach it, since then you have a single function to hide the different parts.